lib/phoenix_live_view.ex

```diff
@@ -551,10 +551,8 @@ defmodule Phoenix.LiveView do
   object's metadata. For example, pressing the Escape key looks like this:
```

Comment:
It might not be a terrible idea to put this link in the documentation.  Would help guide ppl to not manually finding each one and might save some ppl some time

https://w3c.github.io/uievents/tools/key-event-viewer.html

Reply:
I went back and forth if we should tell ppl here as well.  But perhaps a changelog is enough and we don't need to clutter the documentation here that they are deprecated.  Thoughts?

Reply:
Think this is worth considering in another PR. I like the tool, it's pretty useful.